extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/graal/ConfigurationInfoSubstitution.java

```diff
@@ -1,32 +1,26 @@
 package io.quarkus.narayana.jta.runtime.graal;
 
 import com.arjuna.common.util.ConfigurationInfo;
-import com.oracle.svm.core.annotate.Alias;
 import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
 
 @TargetClass(ConfigurationInfo.class)
 final class ConfigurationInfoSubstitution {
 
-    @Alias
-    private static String sourceId;
-    @Alias
-    private static String propertiesFile;
-    @Alias
-    private static String buildId;
-    @Alias
-    private static boolean isInitialized = false;
-
-    // initialize build time properties from data in the jar's META-INF/MANIFEST.MF
```

Comment:
I would keep that comment. And the values probably need to be updated with the values from the new version’s manifest.